src/services/API/setToken.js

```diff
@@ -0,0 +1,17 @@
+import { createAccessTokenEndpoint } from "./endpoints";
+
+export function setToken(apiKey, unique_id) {
```

Comment:
If the arguments are fixed, this function can't be reusable. Every destination does not use `apiKey` and `unique_id`, no.

Reply:
There will be two cases for this:
1) The destination independent of `apiKey` and `unique_id` will not use this function
2) For the destinations having additional fields other than `apiKey` and `unique_id` we can have an argument named data from where apiKey, unique_id and similar other fields can be extracted. Currently, I don't see any destination with such case, but surely It can be modified when we would have such a condition